test/jdk/java/nio/channels/DatagramChannel/SelectWhenRefused.java

```diff
@@ -22,15 +22,17 @@
  */
 
 /* @test
- * @bug 6935563 7044870
+ * @bug 6935563 7044870 8293696
```

Comment:
I think we can omit the bug number in this instance as it is not at this point in time a product bug -- as you are adding diagnostics to the test, maybe a separate subtask or sub-bug could have been used